src/core/util/load_file.cc

```diff
@@ -47,29 +46,43 @@ absl::StatusOr<Slice> LoadFile(const std::string& filename,
 
   file = fopen(filename.c_str(), "rb");
   if (file == nullptr) {
-    error = absl::InternalError(
+    return absl::InternalError(
         absl::StrCat("Failed to load file: ", filename,
                      " due to error(fdopen): ", strerror(errno)));
-    return error;
   }
-  fseek(file, 0, SEEK_END);
+  if (fseek(file, 0, SEEK_END) < 0) {
+    return absl::InternalError(
+        absl::StrCat("Failed to load file: ", filename,
+                     " due to error(fseek): ", strerror(errno)));
```

Comment:
Consider using `absl::StrError` or one of the other wrappers. `strerror` is not thread-safe.

Reply:
I didn't know that, thanks!